### ISSUE TITLE
why I can't use integer in model's name ?

### ISSUE THREAD

**Issue by avilleret** [jamoma/JamomaMax#834]
I have several instance of my model `buffer` that want to access by number like `buffer/1`.
To do so I have to instantiate my model with `[buffer.model "1"]`' and to force symbol casting with quotes.
Would it be possible to cast that internally ? Thus we don't need quote anymore ?
Or is there another issue I'm not aware of ?

This is not a big deal but automatic casting will make it more user friendly.

What do you think ?

**Comment by bltzr** [comment]
Maybe I didn’t get your question, but why don’t you just use instances ? e.g. by calling your models buffer.1, buffer.2, and so on... 
This would allow you to benefit of instance management… (e.g. j.remote_array)

**Comment by avilleret** [comment]
thanks for the workaround.
but I will appreciate to be able to put an integer or even a float as name.
or at least the warning should be more explicit

**Comment by bltzr** [comment]
Sure

**Comment by lossius** [comment]
I agree with Pascal that instances is the way to go here. We could do with a better error message though.

**Comment by bltzr** [comment]
I'm happy that you agree with me, but I don't have anything against it working either... so if someone knows/has the time to do it, I still think it would be good that it also works this way... 
maybe let's leave this issue open, so people stumbling upon this can understand why integer-names don't work, and how to work it around...
